Makefile

```diff
@@ -19,7 +27,10 @@ test-layer1:
 	make -C layer1 test
 	make -C integration_tests test
```

Comment:
I saw it was deleted.

Reply:
Yes my mistake, fixed on the last commit.